evm/scripts/deploy/token.js

```diff
@@ -0,0 +1,14 @@
+async function main() {
+    const Token = await ethers.getContractFactory("Token");
+    const token = await Token.deploy(1*10**8);
```

Comment:
do you mean to do this? This supply is very small atm.
```suggestion
    const token = await Token.deploy(1*10**18);
```

Reply:
looks like that will overflow.  `ethers.utils.parseEther("10").toString()`?

Reply:
`ethers.utils.parseEther("10").toString()` works but also required updating the tasks as the number overflows `toNumber()`